zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/PooledRemoteClient.java

```diff
@@ -44,6 +45,7 @@ public PooledRemoteClient(SupplierWithIO<T> supplier, int connectionPoolSize) {
     GenericObjectPoolConfig poolConfig = new GenericObjectPoolConfig();
     poolConfig.setMaxTotal(connectionPoolSize);
     poolConfig.setMaxIdle(connectionPoolSize);
+    poolConfig.setEvictionPolicyClassName(DefaultEvictionPolicy.class.getName());
```

Comment:
It's similar to the above case. We can add ignore rule for shade as well but I think it's better to set up the class name explicitly

```
Error: Exception in thread "main" java.lang.IllegalArgumentException: Unable to create EvictionPolicy instance of type org/apache/zeppelin/shaded/org.apache.zeppelin.shaded.org.apache.commons.pool2.impl.DefaultEvictionPolicy
```